activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -855,6 +855,7 @@ def remove_reference(table_name, ref_name, foreign_key: false, polymorphic: fals
           else
             foreign_key_options = { to_table: reference_name }
           end
+          foreign_key_options[:column] ||= "#{ref_name}_id"
```

Comment:
Can you give context as to what this is doing and why it's needed? The code prior to the referenced commit didn't include this either, why was it working before?